src/components/Trailer.js

```diff
@@ -1,14 +1,27 @@
 import React from 'react';
 import '../css/Trailer.css';
 
-const Trailer = ({src}) => {
-    return (
-        <iframe id="trailer"
-        className='movie-trailer'
-        title="movie-trailer"
-        src={src}>
-        </iframe>
-    )
+const Trailer = ({backdropImg, src}) => {
+  let content;
+  if (src === 'https://www.youtube.com/embed/') {
+    content = <img
+      className='movie-trailer'
+      src={backdropImg}
+      alt='movie content'
+    />;
```

Comment:
Good logic to alternatively render the backdrop image if we don't have a trailer.